src/archivist/extract/index.js

```diff
@@ -63,7 +69,7 @@ export async function extractFromHTML(sourceDocument) {
       });
       /* eslint-enable no-await-in-loop */
     } catch (error) {
-      throw new InaccessibleContentError(`The filter function "${filterFunction.name}" failed: ${error}`);
+      throw new Error(`The filter function "${filterFunction.name}" failed: ${error}`);
```

Comment:
> I see the point of a higher-order try / catch to wrap the error, but I am not sure I understand why we decrease the specificity of the error here. What do we gain from changing to the generic `Error` type?

Reply:
Following sync conversation: we gain a slight decrease in programmer's necessary attention. I personally don't believe this is worth the churn nor the loss in precision, but I'm fine with it 😉